pkg/scheduler/core/generic_scheduler.go

```diff
@@ -740,8 +740,12 @@ func PrioritizeNodes(
 	for i := range priorityConfigs {
 		if priorityConfigs[i].Function != nil {
 			wg.Add(1)
+			metrics.SchedulerGoroutines.WithLabelValues("prioritizing").Inc()
```

Comment:
Note that the `workqueue.ParallelizeUntil` statement below also creates goroutines. I think you can consider recording the metrics in `workqueue.ParallelizeUntil` to capture all invocations to it.

Reply:
Feel free to defer this.

Reply:
In fact, I think this is not appropriate. `workqueue.ParallelizeUntil` has been set to only use no more than 16 goroutines. If the metrics operation itself increases in intrusion, let alone use the defer operation. 

In general, we need to try to keep the minimum effective calculations and reduce the secondary operations. We can do this by something like `metrics.SchedulerGoroutines.WithLabelValues("prioritizing").Inc(16)`, or ignore it.